buildpipeline/pipeline.json

```diff
@@ -147,6 +147,32 @@
       "DependsOn": [
         "Trusted-All-Release"
       ]
+    },
+    {
+      "Name": "Signing validation",
+      "Parameters": {
+        "TreatWarningsAsErrors": "false"
+      },
+      "BuildParameters": {
+        "BuildConfiguration": "Release"
+      },
+      "Definitions": [
+        {
+          "Name": "Core-Setup-Signing-Validation",
```

Comment:
This is consistent with the other definitions in the branch, and it makes it easy to see what repo a build references or to search for it in `DEVDIV.visualstudio.com`.